ee/webhooks/cmd/flag/flags.go

```diff
@@ -32,21 +36,24 @@ const (
 )
 
 var (
-	DefaultRetriesSchedule = []time.Duration{time.Minute, 5 * time.Minute, 30 * time.Minute, 5 * time.Hour, 24 * time.Hour}
-	DefaultRetriesCron     = time.Minute
+	DefaultRetriesCron = time.Minute
 )
 
 func Init(flagSet *pflag.FlagSet) {
 	flagSet.Bool(Debug, false, "Debug mode")
 	flagSet.String(LogLevel, logrus.InfoLevel.String(), "Log level")
 
 	flagSet.String(Listen, DefaultBindAddressServer, "server HTTP bind address")
-	flagSet.DurationSlice(RetriesSchedule, DefaultRetriesSchedule, "worker retries schedule")
+	flagSet.DurationSlice(RetriesSchedule, nil, "worker retries schedule")
```

Comment:
You can drop the flag, the service is not tab stable today, and we use env vars to configure services.